include/smack/GenModifies.h

```diff
@@ -0,0 +1,73 @@
+//
+// This file is distributed under the MIT License. See LICENSE for details.
+//
+#ifndef MODS_H
```

Comment:
Change the file name to `ModAnalysis.h`.
Also, this should be `#ifndef MODANALYSIS_H` to make it consistent with other files.